postman/generate_collection

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+. bin/util
+
+# bundle spec
+announce "Bundling Spec File"
+bin/bundle_spec
+
+announce "Generating Postman Collection"
+docker run --rm \
+    -v ${PWD}:/openapi_spec \
+    node:latest \
+    /bin/bash -c "npm i -g openapi-to-postmanv2
+    openapi2postmanv2 -s /openapi_spec/spec.yml -o /openapi_spec/postman/collection.json -p -O collapseFolders=false,folderStrategy=Tags"
```

Comment:
nit: I think we should be outputting to the same folder as the openapi spec generator, maybe with a subfolder? So we output to `out/postman/collection.json`.